vms/platformvm/client_permissionless_validator.go

```diff
@@ -39,13 +39,14 @@ type ClientOwner struct {
 // over client
 type ClientPermissionlessValidator struct {
 	ClientStaker
-	ValidationRewardOwner *ClientOwner
-	DelegationRewardOwner *ClientOwner
-	PotentialReward       *uint64
-	DelegationFee         float32
-	Uptime                *float32
-	Connected             *bool
-	Signer                *signer.ProofOfPossession
+	ValidationRewardOwner  *ClientOwner
+	DelegationRewardOwner  *ClientOwner
+	PotentialReward        *uint64
+	AccruedDelegateeReward *uint64
```

Comment:
I think we need to populate this in `getClientPermissionlessValidators`

Reply:
[1df7471](https://github.com/ava-labs/avalanchego/pull/1291/commits/1df747180c81060f4834bd388b43ce396557f18b)